### PR TITLE
Gseal 1 initial front end layout

### PR DESCRIPTION
**Pull request by gpseal**
Initial layout for the front end.  Not everything functions correctly at this stage and parts will need to be refactored, but it is ready to have API data applied to it and we can work out other bits and pieces once that is all in order.
Story links don't work correctly currently, if they do, they will link to the news topic page.

### CODE REVIEW

File: client/src/pages/Home.tsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    category: string
+}
+
+const Home = (props: Props) => {
+
+    const [data, setData] = useState<any>();
+        
+        const getData = async () => {
+            try {
+                const response = await axios.get("http://localhost:3002/article.list");
```
Comment (AardhynLavender) [inline client/src/pages/Home.tsx:37]: Can we store the domain for the server in an environment variable. Vite should automatically load in variables from `.env` files if the variable is prefixed with `VITE_`

https://vitejs.dev/guide/env-and-mode.html#env-files

```env
# client/.env
VITE_SERVER_DOMAIN = http://localhost:3001/ # or 3002 😉
```

```typescript
// someReactFile.tsx
const domain = import.meta.VITE_SERVER_DOMAIN
```

**Review by AardhynLavender — CHANGES_REQUESTED** [review]
Looks pretty good. Can't wait for some gardening news!

File: client/src/pages/Home.tsx (hunk 1 above)
Comment (gpseal) [reply client/src/pages/Home.tsx]: Good idea, have updated to include this

**Review by AardhynLavender — APPROVED** [review]
Nice.

**Comment by ArtemKech** [comment]
Looks to good to be a prototype! I like the way you emphasized AI in "DAILY". Looks nice and neat. Good job

**Pull request merged**